crossplane-provider-gcp.yaml

```diff
@@ -27,27 +27,25 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      repository: https://github.com/upbound/provider-gcp
+      repository: https://github.com/crossplane-contrib/provider-upjet-gcp
       tag: v${{package.version}}
-      expected-commit: d1d5a0ec3b200cdfbd67f67de7040e7f29427c62
+      expected-commit: 6738b686330f7ec2f4037a200e2718fa3628243e
 
-  - runs: |
-      # `make` downloads `up`, unless we move our prebuilt `up` to where it expects it.
-      GOARCH=$(go env GOARCH)
-      UP_VERSION=$(up version --client=true --format json | jq .client.version)
-      UP_VERSION="${UP_VERSION#*: }"
-      echo "Using up ${UP_VERSION}"
+  - uses: patch
+    with:
+      # not working with UP_UPTEST_LOCAL variable in Makefile
```

Comment:
it's probably worth mentioning what's not working, also any chance the reason it's not working is a genuine issue?

Reply:
A change was made to the repository from which this package is obtained. The current package is installed with the UP command in version 0.28.0. We currently have the package around version 0.32.0, because of this the repository installer package looks for the version of UP that had previously been passed as an argument, however, it is looking for a version of a variable that does not exist in the current repository. Should I add the comment into the code?

Reply:
Trying to be more detail 

A change was made to the repository link, version 0.28.0 is specifically used to build the package. When adding the Wolfi package it is in version 0.30.0 which causes problems with the repository variables, previously (the previous repository) the UP_VERSION variable was overwritten, in the current repository it causes a problem with the version in addition to being necessary Pass the variable UPTEST_LOCAL_VERSION which has the current value v0.12.0-9.gac371c9 which is not counted for version 0.30.0.
In addition to this, the repository installs version 0.28.0 which is compatible.

Reply:
According to the [PR](https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/Makefile) for aws operator, patching this file it could be fixed.